include/chaiscript/language/chaiscript_engine.hpp

```diff
@@ -52,6 +52,7 @@
 
 
 #include "../dispatchkit/exception_specification.hpp"
+#include "chaiscript_parser.hpp"
```

Comment:
Still not sure it's a good idea to pull in the parser for this very simple test one could easily inline, especially considering it's only used in one place.

Reply:
That's true, I originally intended it to be done differently... I've refactored the code and removed the include.